### ISSUE TITLE
IconMenu not working

### ISSUE THREAD

**Issue by VinceBT** [markoshust/meteor-material-ui#1]
Hi I am absolutely grateful that you have developed this meteor package :)
I am using it right now in an app I'm creating for android.

The problem comes from a  component : 
I have this JSX code : 

``` jsx
<IconMenu iconButtonElement={<IconButton><NavigationMoreVert /></IconButton>}>
   <MenuItem index={1} primaryText="Example 1" />
   <MenuItem index={2} primaryText="Example 2" />
   <MenuItem index={3} primaryText="Example 3" />
</IconMenu>
```

And it doesnt do anything when I click on it : 

![8117231e4e891c8f3fdc9ad6531f40c9](https://cloud.githubusercontent.com/assets/1362505/11001045/6d3d937e-84a5-11e5-9f44-1cc936096d70.png)

It is a problem coming from your repo ?

Thanks by advance for any help and thank you again for the port of this package to meteor !

**Comment by markshust** [comment]
Hi @BTVincent -- glad you find it useful! I was searching and searching for a package like this myself, and nothing existed that was fairly updated :) I plan on using material design a lot in the future, so I will absolutely be supporting this package.

I'm not too familiar with mui yet, but I would assume your code should work. Let me look into this, as I want to do the exact same component setup myself :)

**Comment by VinceBT** [comment]
I was looking for a package like this but all the packages were abandoned :( 

I am happy that you are active and determinate to improve your package to the best !

I have found this issues on the main package if you need help : 

https://github.com/callemall/material-ui/issues/1346
https://github.com/callemall/material-ui/issues/1797

Keep up the good work :) !

**Comment by VinceBT** [comment]
I opened an issue on their repo because I think the problem comes from them : 

https://github.com/callemall/material-ui/issues/2090

**Comment by markshust** [comment]
Thank you -- this is/was an issue with their repo, however they provided a possible fix for using their current implementation of menu components. I added the temp workaround and exposed it as: `mui.Menus`.

So you can now use the older (current) implementation of the MenuItem component by referencing `mui.Menus.MenuItem`. If you reference `mui.MenuItem`, this will pull the newer component implementation they are working on.

Anywho, I updated the README to reflect this usage, and it all appears working now :) Released `0.13.1_4` which contains this and all the other fixes I just complete (some bug fixes, etc).
